cpp/src/groupby/common/aggregation_requests.hpp

```diff
@@ -52,6 +53,44 @@ using AggRequestType = std::pair<gdf_column*, operators>;
  *---------------------------------------------------------------------------**/
 using SimpleAggRequestCounter = std::pair<AggRequestType, gdf_size_type>;
 
+
+static constexpr std::array<operators, 4> simple_aggregations = {SUM, MIN, MAX, COUNT};
+
+static constexpr std::array<operators, 2> ordered_aggregations = {MEDIAN, QUANTILE};
+
+namespace constexpr_util {
```

Comment:
This isn't used outside this file, so I would just put it in an anonymous namespace.  And I think you can move `simple_aggregations` and `ordered_aggregations` into the namespace?

Reply:
Anonymous namespaces shouldn't be used in headers.

Reply:
All right!